targets/XPlatCppSdk/source/.gitignore.ejs

```diff
@@ -304,7 +307,24 @@ external/openssl-build-debug/**/*
 !external/openssl-build-debug/makefile
 !external/openssl-build-debug/readme.txt
 
-# Linux Build
+# Json xdk build
+external/jsoncpp-xdk-build/Durango
+external/jsoncpp-xdk-build/Debug
+external/jsoncpp-xdk-build/Profile
+external/jsoncpp-xdk-build/Release
 
+# Linux Build
 build/Linux/*.o
-build/Linux/XPlatCppLinux
\ No newline at end of file
+build/Linux/XPlatCppLinux
+
+# Xbox build generated files
+build/Xbox/durango
```

Comment:
>d [](start = 11, length = 1)

I'd recommend to use [Dd] syntax (for all directories created by Visual Studio). I.e. [Dd]ebug, [Rr]elease, etc. Cases may change in some weird situations